### DISCUSSION TITLE
Adjusted Content Release Order for TBC

### DISCUSSION THREAD

**Discussion by balleny** [chromiecraft/chromiecraft#4146] [Suggestions]
To cut it short: Consider releasing the raids in terms of their difficulty (or as an easier indicator: by the ilvl of the drops) instead of the original release schedule.

We just experienced, how annoying it is to run BWL for so long as the "primary" raid. ZG and AQ20 had been designed as easy catch-up raids for lategame joiners. Here, it results in people being mostly bored by the new content (as the average joe is already overgeared for the content.

IMHO it would be way nicer to get Zul'Aman released between t5 and t6, instead of having it released before/with SWP as per current Progression timeline [https://www.chromiecraft.com/en/progression/](https://www.chromiecraft.com/en/progression/)

(similar argument could be made with access to the sunwell isle (not the raid) and Magisters Terrace normal and heroic mode, as well as the token gear (here I would suggest to delay token gear release e.g. 4 weeks, as people will otherwise directly access and buy everything at day 0)

**Comment by djobf** [comment]
Agreed. Might spark some backlash but I'm sure in the long run everyone will realize it's for the better

**Comment by Cooties-cc** [comment]
Super hot take : ZA should be release between T4 and T5. 
Why this crazy notion? It's all about the average ilvl that you can obtain. Here are the stats from each end boss in terms of ilvl. 

Kara - Prince Malchezaar : 125 ilvl 
Gruul's Lair - Gruul the Dragonkiller : 125 ilvl 
Magtheridon's Lair - Magtheridon : 125 ilvl 

All the end bosses of T4 drop ilvl 125 max. 

Zul'aman - Zul'jin : 133-138 ilvl

The end ilvl of ZA is avg 135.5 which puts it smack between T4 and T5, since T5 values are...

Serpentshrine Cavern - Lady Vashj : 138-141 ilvl
The Eye - Kael'thas Sunstrider : 138-141 ilvl

T5 has an avg ilvl of 139.5 which puts it above ZA. This is also to say some crafted gear has higher ilvl on avg than ZA drops around the start of T5, where nether vortexes are obtained. Take a look at BS craft Bloodmoon, which is ilvl 136. 

If ZA was to be released alongside SWP, absolutely no one would be doing the raid except for the mount. You would have far fewer reports being made about the bugs in there to be fixed. It's not in line with how blizz released content, but blizz is also known for doing regretful things. 

I stand by my vote to release ZA after T4 has been completed.

**Reply by balleny** [reply]
going strict by itemlevel, i would agree.
but, releasing 10 person content (with TBC defaults and wotlk content) prior to t5, would somehow diminish the value of 25 player raids quite a bit. Given, that it was intended as a catchup raid with an additional challenge part for the back then "hardcore" raiders, I would prefer it between t5 and t6. Also, that way you have some 10 player raid releasing later on as alternative to 25 raids only (which would be the case if we go t5 -> t6 -> sunwell)

**Reply by Cooties-cc** [reply]
I still stand with the post T4 release. It also goes alongside the purpose of the server, which is to test everything and fix problems. If the raid was released post T5, it would get far less traffic from players doing the raid, since the ilvl decreased value in drops. It can still be considered a catchup raid post T5 for players without a guild, or coming into a new guild. Of course this basis of logic is by what gear upgrades/value you get per raid, and nothing else. ZA could hold a bis item over a T5 raid (hadn't checked, but sorta doubt), which was an issue for vanilla content. Here's a comparison breakdown for vanilla content if valuing ilvl over how blizz released content :

Blizz order (server order) : 
MC : 73-78 (75.5)
ony : 71-76 (73.5)
azure : 66-71 (68.5)
kaz : 66-71 (68.5)
bwl : 76-83 (79.5)
zg : 68
wb drags : 71-72 (71.5)
aq20 : 72
aq40 : 84-88 (86)

How it should be if going by ilvl :
ZG
azure/kaz
ony
MC
BWL
WB drags
aq20
aq40

I wouldn't mind how the ilvl order goes by raids, since as a raid comes out, you're getting stronger items, instead of weaker items. In an RPG aspect, I find this more appropriate, but others disdain from going off the path of how Blizzard originally paved out content. Of course the idea of the server is still to fully test content to have it fixed, so priority should still go with highest testing base. I don't really care how it goes either way, but just giving my perspective on making it smoother or more enjoyable.

**Reply by Tereneckla** [reply]
Going strictly by item level is the wrong way to judge the power of the items.
Being released after T6 means the ZA items have a way better stat budget than T5 (also haste and ARP)
Many items from ZA are bis even in T6

**Reply by balleny** [reply]
which wont invalidate the other reasons that speak for an alternating release of 25s and 10s

**Reply by Tereneckla** [reply]
Well, this sub-discussion is for ZA between T4 and T5 which is unthinkable.
Between T5 and T6 also is not ideal imo, but acceptable
Overall SWP was meant as a challenge for the people decked out in T6 gear (which was because of the long T6 phase, 2.1 to 2.3)
For current standards untuned SWP won't be a challenge for a regular 3 month raid cycle, but going by the spirit of it I advocate for the original release order

**Reply by ldzetc** [reply]
I think that the item level is deceiving.  Item level wasn't really a thing until wrath and progression wise it doesn't fit inbetween T4 and T5.  People are still farming kara pretty regularly.  
After T5, most kara gear becomes less important and I think adding ZA between T5 and T6 would help in a number of ways with content, release schedule, and player satisfaction.  
After T6 and before Sunwell, we have heroic MGT and sunwell dailies to do as well as more raid content in general.  I think being mostly finished with ZA by the time sunwell is coming out is pretty solid.
Also, ZA introduces a lot of valuable haste gear which isn't available until T6 and Sunwell.  It would give a moderate headstart,for a Pre-T6 release, but doing it before T5 doesn't feel right.  Haste availability on gear has a much bigger impact than iLevel.

**Reply by balleny** [reply]
People kept running Kara in the later stages especially on new chars for the easily obtainable Badges compared to heroics, too

**Comment by balleny** [comment]
unfortunately, we are still lacking any kind of information regarding the access to pvp items in the sense of pre-patch style.
aka when will the the restrictions been lifted? lvl 69 bracket, lvl 70 bracket?

Also, we dont have any details if the removal of world buffs once entering raids, will be dropped and when (considering the world buffs are only applicable up to lvl 63, it would be nice to have the possibility to do 1-2 raids with world buffs

**Comment by balleny** [comment]
### Proposal for releases
obviously not complete, but offering some hints for what to pay attention to:

#### preparations beforehand to prevent early bird abusage
- increase reputation requirements to buy heroic access keys back to revered
- increase reputation requirements to buy reputation based pvp gear back to revered
- remove arena gear vendors
- remove level 70 honor gear vendors


#### Level 61-64
- Access to: Hellfire Ramparts, Blood Furnace, Slave Pens, Underbog, Mana-Tombs, Auchenai Crypts
- MC & ZG get their tuning removed
- New Battleground: Eye of the Storm

#### Level 65-69
- Access to: The Escape From Durnholde, Sethekk Halls, Shadow Labyrinth, Shattered Halls, The Arcatraz, The Black Morass, The Botanica, The Mechanar, The Steamvault
- BWL, AQ20 get their tuning removed
- Level 60 pvp items will become available as it is blizzlike on Wotlk.

#### Level 70 Pre-Tier
- all heroics become accessible except for Isle of Quel'Danas
- Heroic bosses drop badges of Justice
- Badge of Justice vendors sell all items below itemlvl 111
- badge of justice vendors DONT sell Primal Nether, Nether Vortex or Gemstones
- Aq40 tuning removed
- itemlevel 115 PvP Trinkets as well as everything listed in the link becomes available, **besides the gems**: https://www.wowhead.com/tbc/guide/burning-crusade-classic-phase-1-battleground-pvp-rewards-and-costs-13292
- Spawning of Kazzak and Doomwalker world bosses

#### Level 70 Tier 4
- karazhan, mag and gruul accessible
- Raid bosses DONT drop badges of justice
- no change to badges of justice vendors
- Arena season 1 available including the Arena Set 1
- pvp honor vendor starts offering "General's"
- exchange vendor to retrieve arena gear for pve tokens is accessible (blizzard also introduced it early in classic tbc compared to tbc)

#### Level 70 Tier 5
- SSC and TK available, bosses dont drop badges
- arena season 2
- honor vendors offer the "Veterans" pvp items
- no change to badge vendors

#### level 70 ZA
- ZA becomes available
- ZA related badge vendor loot (ilvl 128) becomes available
- raid bosses start dropping badges of justice
- honor vendors also start selling gems
- (mabye not a new arena season but a arena tournament?)

#### Level 70 T6
- access to BT, Hyjal AND Isle of Quel'Danas, including the 5s dungeon, but not Sunwell
- No access to Sunwell Isle Badge loot
- Arena season 3
- Honor pvp vendors offer "Vindicator" Gear (ilvl 141)

#### Level 70 Sunwell
- Access to sunwell
- badge vendors offer everything they are supposed to
- Arena season 4
- honor vendors offer "Guardians" gear (ilvl 154)

**Reply by balleny** [reply]
#### Further points to consider
- Primal Nether used to be bind on pickup until ZA release, will this be reintroduced?
- Plenty of professions had abilities with an internal cooldown, including Alchemy: Transmute Primal Might and Tailoring Cloths. Will those be introduced back?
- Shattered Halls and Arcatraz required atleast one person to obtain an access key via quest. Will this be a thing again?
- Will RDF respect access requirements for normal modes (Taverns of Time pre-quests, Arcatraz Key, Shattered Halls quest)?
- Skettis, Ogrila and the Netherwing dragons areas / quests accessibility
- Kazzak and Doomwalker: accessible with which phase / contested spawn / current system?
- in TBC pvp items required a mix of pvp tokens and honor to buy, will we see this being re-introduced?

those are just further points to be considered.

**Reply by Tereneckla** [reply]
I mostly agree with this proposal.
One thing I would change is splitting your Level 70 T6 in two
Have BT/Hyjal in one release and Isle of Quel'Danas + MGT at some point before SWP

**Reply by balleny** [reply]
the idea was mostly to give people that arent too much into raiding something to do, too.
also, if we do Quel'Danas, there was a War Effort like system involved, which would require time and would release content only gradually

**Comment by Amrx2** [comment]
Hi balleny, we have discussed this one at length with the whole staff team, as well as the admins regarding this. Firstly, we have already published when each tuning will be dropping off vanilla raids, you can find that here - https://www.chromiecraft.com/en/outlands-14th-december/

In terms of the raid releases, we also will be keeping it the same as we currently have. Whilst in terms purely from a gear progression standpoint it would make sense, that would however invalidate the reasons of the raids existing in the first place, that being for new raiders getting up to speed, getting some good gear before tackling 'proper' raids (i.e your Black Temples, Sunwell etc). ZG will be run still as you said, for the mount more than anything at every opportunity so reports will continue to come in (even now on Vanilla taking a 7 day average to cancel out the shorter raid lockouts, ZG is by far the most popular raid on CC. Those mounts are in demand!).  

Professions will receive a dedicated section in our 'How it will work in TBC' post that will come soon as well. 

Outlands PvP gear (including from Arenas) won't be available at the start either, R14 only for the time being and that will be covered in the post as well. Purchasing methods will also be mentioned with probably a token system in place once again (to be confirmed however).

We will go into more detail regarding dungeons with pre-requisite quests as well, such as the Caverns of Time ones. This might mean a change in the release of some dungeons of course, discussing it with the team now. 

Expect further clarification on these in the coming week or two.

Thank you for the lovely post and all concerns raised, have a good day!
Construct

**Comment by Blessed-Sunfruit** [comment]
ZA before BT is a great idea, having ZA after BT means 2-3 months of people getting bored because no progression if you already have BT on farm, then bracket planned after ZA is isle of quel'danas and a dungeon release, so another 2-3 more months of no new raid for mains **we're talking up to 9 months of BT farm** by the time Sunwell is out - please consider this before it's too late!!!

**Reply by balleny** [reply]
in the end it might end up being BWL/ZG situation all over again

**Comment by Amrx2** [comment]
Duplicate of [6171](https://github.com/chromiecraft/chromiecraft/discussions/6171), position remains unchanged and given devs focus on future content out of CC and roadmap in place, alterations not being made.
